pennylane/qnn/torch.py

```diff
@@ -150,6 +150,30 @@ def qnode(inputs, weights_0, weights_1, weights_2, weight_3, weight_4):
 
             qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes, init_method=init_method)
 
+        **Model saving**
+
+        Instances of ``TorchLayer`` can be saved using the usual ``torch.save()`` utility:
+
+        .. code-block::
+
+            qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes)
+            torch.save(qlayer.state_dict(), SAVE_PATH)
```

Comment:
@eddddddy, could we have an example that includes a model with multiple layers? E.g.
```suggestion
            qlayer = qml.qnn.TorchLayer(qnode, weight_shapes=weight_shapes)
            clayer = torch.nn.Linear(2, 2)
            model = torch.nn.Sequential(clayer, qlayer)
            torch.save(model.state_dict(), SAVE_PATH)
```

Reply:
And maybe we have this in the tests?

Reply:
Added another example in the docstring on top of the previous one. Also changed the test.